### PR TITLE
chore: pnpm cleanup tasks: prune, dedupe

### PR DESCRIPTION
**Pull request by tyleroooo**
These should make other changes easier and maybe even reduce bundle size.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -165,7 +165,7 @@
     "eslint-import-resolver-typescript": "^3.5.5",
     "eslint-plugin-import": "^2.27.5",
     "eslint-plugin-jsx-a11y": "^6.7.1",
-    "eslint-plugin-n": "^16.0.0",
+    "eslint-plugin-n": "^15.7.0",
```
Comment (tyleroooo) [inline package.json:168]: was causing peer dep failure

File: package.json [hunk 2]
```diff
@@ -176,6 +176,7 @@
     "pnpm": "^8.6.6",
     "prettier": "^2.8.8",
     "rollup-plugin-sourcemaps": "^0.6.3",
+    "stream-browserify": "^3.0.0",
```
Comment (tyleroooo) [inline package.json:179]: We have a vite fallback but for some reason weren't requiring this so dev/build were failing

File: src/lib/abacus/websocket.ts [hunk 3]
```diff
@@ -20,9 +20,9 @@ class AbacusWebsocket implements Omit<AbacusWebsocketProtocol, '__doNotUseOrImpl
 
   private receivedCallback: ((p0: string) => void) | null = null;
 
-  private disconnectTimer?: NodeJS.Timer;
+  private disconnectTimer?: NodeJS.Timeout;
 
-  private reconnectTimer?: NodeJS.Timer;
+  private reconnectTimer?: NodeJS.Timeout;
```
Comment (tyleroooo) [inline src/lib/abacus/websocket.ts:25]: dedupe resolved us to a slightly higher version of node types and apparently these are called timeouts now

**Review by rosepuppy — APPROVED** [review]
Awesome, thanks for this!

**Pull request merged**